### DISCUSSION TITLE
importer des fichiers scorms dans CStudio?

### DISCUSSION THREAD

**Discussion by Moigaston** [chamilo/chamilo-lms#5813] [General]
il n'est pas possible d'importer des fichiers scorms dans CStudio. On ne peut  les y intégrer qu'avec un lien. Exact?

**Comment by Ringoinmel** [comment]
I am also not able to import backup to the course.

**Comment by LudiscapeXApi** [comment]
Je ne comprends pas très bien la question. Ce que je peux vous dire, c'est que CStudio (Chamidoc Studio) permet effectivement de créer des fichiers SCORM, mais ne peux pas les importer directement en tant que projet modifiable. SCORM étant un standard de contenu compilé, il est souvent distribué sous forme de lien ou d'archive. Chamilo LMS permet de lire et d'importer ces fichiers SCORM pour qu'ils soient utilisables dans des modules d'apprentissage.

Si vous cherchez à importer un fichier SCORM dans CStudio, cela peut poser problème, car ce dernier sert principalement à la création de ces fichiers, et non à leur importation ou édition, puisque SCORM est un format compilé et non éditable. Dans ce cas, vous devrez peut-être passer par un lien externe ou un autre système comme Chamilo LMS pour l'importation.

**Comment by Moigaston** [comment]
Merci pour ta réponse, Damien. Je vais réexpliquer de manière plus claire, en précisant que j'ai bien compris qu'il est possible de créer des grains de formation avec Chamidoc Studio (anciennement appelé ChamiloStudio), lesquels peuvent être téléchargés au format SCORM. Ma question est toute autre : Partant du fait que l'outil Parcours de Chamilo me permet d'importer des exercices au format SCORM et que je peux vérifier, grâce à la barre de progression, si mes élèves ont fait ces exercices, ma question est la suivante : Est-il possible, comme dans l'outil Parcours de Chamilo, d'importer des fichiers au format SCORM dans Chamidoc Studio — ce que je ne pense pas —, ou bien peut-on seulement les intégrer à un projet Chamidoc Studiopar un lien vers le fichier SCORM, comme avec LearningApps ? Lorsqu'on utilise seulement un lien, il n'est pas possible de vérifier si les élèves ont fait ou non l'exercice. Voilà pourquoi j'aurais trouvé utile de pouvoir importer des fichiers SCORM directement dans Chamidoc Studio - et d'y trouver une barre de progression :).

**Comment by ywarnier** [comment]
En fait, dans Chamilo et Chamidoc Studio sont deux *types* différents de logiciels : Chamilo est un LMS, CStudio/Chamidoc Studio est un "outil auteur". Comme tous les outils auteur (ou presque), CStudio permet de créer du contenu dans un format interne qui lui est propre, puis d'exporter ce contenu en SCORM. Un LMS agit principalement (dans ce contexte) comme un "lecteur/reproducteur" SCORM, c'est à dire qu'il permet de faire utiliser un même contenu par une grande quantité d'apprenants, et d'en extraire des données de progrès pour ensuite les présenter sous forme analytique, tableaux, etc.

Donc a priori CStudio pourrait (plus tard) se transformer aussi en "lecteur/reproducteur SCORM" en appliquant la partie correspondante de la norme SCORM et ainsi en supportant les protocoles de communication de SCORM au niveau RTE (Real Time Execution engine), mais je ne crois pas que ce soit l'intention.

Damien (l'éditeur de CStudio, par ailleurs) a certainement une réponse plus concrète à ce sujet.
